src/content/en/2020/javascript.md

```diff
@@ -208,33 +216,53 @@ Of those external scripts, only 12.2% of them are loaded with the `async` attrib
   sql_file="breakdown_of_scripts_using_async_defer_module_nomodule.sql"
 ) }}
 
+{# TODO(authors): Why is it misleading? #}
 Considering that `defer` provides us with the best loading performance (by ensuring downloading the script happens in parallel to other work, and execution waits until after the page can be displayed), we would hope to see that percentage a bit higher. In fact, as it is that 6.0% is a bit misleading.
 
 Back when supporting IE8 and IE9 was more common, it was relatively common to use _both_ the `async` and `defer` attributes. With both attributes in place, any browser supporting both will use `async`. IE8 and IE9, which don't support `async` will fall back to defer.
 
+{# TODO(authors): In the Twitter thread that discussed this, it actually spurred the Jetpack folks to push a fix! Might be worth mentioning :) #}
```

Comment:
@tkadlec totally optional

Reply:
Great little story! Added. :)

Reply:
Thanks! Could you open a new PR with your changes?